lib/tasks/deploy.rake

```diff
@@ -73,6 +71,21 @@ namespace :deploy do
     puts "Build URL: #{response['build_url']}"
   end
 
+  def deploy_envs
+    return DEPLOY_ENVS unless ENV.key?('DEPLOY_ENVS')
+    # e.g., DEPLOY_ENVS=master:production,staging:staging,staging:preview
+    ENV.fetch('DEPLOY_ENVS').split(',').each_with_object(Hash.new([])) do |obj, pair|
```

Comment:
Ah, you're right, missed that.  It seems the second variant is workable with e.g. `x[:a] = x[:a].push('bla')`, but I just got rid of the hash default.  I still need to test these changes a bit with one of the apps—as you can see :)